OpenEdXMobile/AndroidManifest.xml

```diff
@@ -219,9 +219,10 @@
 
         <activity
             android:name="org.edx.mobile.view.CertificateActivity"
+            android:exported="true"
```

Comment:
I tried to reproduced the issue but failed to due to unavailability of course with certificate link.

Reply:
I think we won't need the Certificate link because the share button is in the toolbar.
Plus according to the logs, the issue was raised before even the user clicked the share button. 🤔